digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/SpeedLimitService.scala

```diff
@@ -60,7 +60,7 @@ class SpeedLimitService(eventbus: DigiroadEventBus, vvhClient: VVHClient, roadLi
 
     val newSpeedLimits = fillNewRoadLinksWithPreviousSpeedLimitData(projectableTargetRoadLinks,
       oldSpeedLimits ++ speedLimitsOnChangedLinks, speedLimitsOnChangedLinks, change)
-    val speedLimits = (speedLimitLinks.filterNot(sl => newSpeedLimits.map(_.linkId).contains(sl.linkId)) ++ newSpeedLimits).groupBy(_.linkId)
+    val speedLimits = (speedLimitLinks ++ newSpeedLimits).groupBy(_.linkId)
```

Comment:
Filler now takes care of overlapping entries, it's better done there. Previous version had side effects when projected data was older and was reprojected over and over again.